Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
Not necessarily.
Pod install is only needed when you've added a new pod.
https://guides.cocoapods.org/using/pod-install-vs-update.html

Here's a failing job: 
https://travis-ci.org/google/EarlGrey/jobs/221570274